### PR TITLE
Adding 'Getting Started' guide and a basic FAQ with 5 questions (for now)

### LINKED ISSUE #17: Write out Getting Started Guide for the website.
This should be more extensive than what we have on Github. Basically, assume the customer knows very little about WoTT and this should take them from zero to a fully installed WoTT agent claimed in the dashboard. We should also cover the basics of the device audit in the console and how to manage credentials.

### PR DESCRIPTION
**Pull request by fshmcallister**
Getting started:

- includes introduction to WoTT in a more conversational tone than my other documentation
- discusses cryptographic identity + credential management (this may get moved to another document later - tbd)
- instructions on installing agent and registering to dash
- closes #17 

FAQ: 

- brief FAQ set up using indexed hyperlink layout
- includes 5 questions I think may be relevant to first time WoTT users
- still leaving in progress as will likely need more altering

### CODE REVIEW

File: documentation/faq/README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Frequently asked questions
+
+- [What Operating Systems does the WoTT agent support?](#1)
```
Comment (vpetersson) [inline documentation/faq/README.md]: @fshmcallister these anchor links are important for SEO, so please name them something more descriptive than numbers.

File: documentation/getting_started/README.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
+# Getting Started
+
+Welcome to WoTT - the first step to securing your IoT devices.
+
+## A brief overview of WoTT's services
+
+WoTT's primary goal is to simplify and manage end-to-end security for IoT developers. 
+We provide this in a number of ways:
+
+- Simplified encryption of device communication
```
Comment (vpetersson) [inline documentation/getting_started/README.md]: This is just minor stylistic preference, but let's use `*` for bullet points.

File: documentation/getting_started/README.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+- Cryptographic identity (verification of access)
+- Enabling the removal of hard coded credentials from web applications and firmware
+
+You can access all these features by installing the [WoTT Agent](https://github.com/WoTTsecurity/agent) and using the [WoTT IO dash](https://dash.wott.io) to manage your devices through a simple and user-friendly interface. We'll walk you through step-by-step in setting up and installing the WoTT agent so you can start to implement hassle-free security on your IoT devices.
```
Comment (vpetersson) [inline documentation/getting_started/README.md]: Let's refer to the dashboard as "WoTT's Dashboard"

Comment (fshmcallister) [reply documentation/getting_started/README.md]: Fixed
I missed this one oops, thanks for pointing it out.

File: documentation/getting_started/README.md (hunk 2 above)
Comment (fshmcallister) [reply documentation/getting_started/README.md]: Fixed

File: documentation/faq/README.md (hunk 1 above)
Comment (fshmcallister) [reply documentation/faq/README.md]: changed to be more descriptive

Comment (fshmcallister) [reply documentation/faq/README.md]: Let me know if you can think of any other pressing questions

File: documentation/getting_started/README.md [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+
+## Managing your own credentials through the WoTT Dashboard
+
+In scenarios where you might require API keys or logins for your applications, you can add and manage these credentials to the WoTT Dashboard. On the left of WoTT Dashboard is a sidebar.
+
+![sidebar](https://github.com/WoTTsecurity/wott-io/blob/master/documentation/getting_started/images/sidebar.png)
+
+Navigate to 'Credentials.' Select 'Add a credential'
+
+![credentials](https://github.com/WoTTsecurity/wott-io/blob/master/documentation/getting_started/images/credentials.png)
```
Comment (vpetersson) [inline documentation/getting_started/README.md]: So these need to be added in a slightly different fashion for the website. I realized I never told you that. You can find an example [here](https://raw.githubusercontent.com/WoTTsecurity/wott-io/master/_posts/2019-06-25-why-open-source-solutions-are-critical-for-IoT.md), but in short:

```
{% asset blog/iot-monster.png srcset:width="1300 2x" srcset:width="650 1x" alt="No Image" class="img-fluid" %}
```

(Where you need to tweak the variables.)

File: documentation/getting_started/README.md (hunk 2 above)
Comment (vpetersson) [reply documentation/getting_started/README.md]: Probably failed to commit that part as it still shows `-`.

**Pull request merged**